documentation/modules/cruise-control/proc-cruise-control-topic-replication.adoc

```diff
@@ -0,0 +1,142 @@
+You can adjust the replication factor for specific purposes, such as:
+
+* Setting a lower replication factor for non-critical topics or because of resource shortages
+* Setting a higher replication factor to improve data durability and fault tolerance
+
+The Topic Operator uses Cruise Control to make the necessary changes, so Cruise Control must be deployed with Strimzi.
+
+The Topic Operator watches and periodically reconciles all managed and unpaused `KafkaTopic` resources to detect changes to `.spec.replicas` configuration by comparing the replication factor of the Kafka topic in Kafka.
+One or more replication factor updates are then sent to Cruise Control for processing in a single request.
+Replication factor changes are handled by the following Cruise Control operations:
+
+* Requests to modify the replication factor are made through Cruise Control's `/topic_configuration` endpoint
+* Checks on the progress of the changes are made through Cruise Control's `/user_tasks` endpoint
+
+Progress is reflected in the status of the `KafkaTopic` resource.
+
+NOTE: This operation requires the Topic Operator to be working in unidirectional mode. It does not work when using the Topic Operator in bidrectional mode.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+* xref:deploying-the-topic-operator-using-the-cluster-operator-str[The Topic Operator must be deployed] and running in unidirectional mode to manage topics through the `KafkaTopic` custom resource. 
+* xref:proc-configuring-deploying-cruise-control-str[Cruise Control is deployed with Kafka.]
+
+.Procedure
+
+. Edit the `KafkaTopic` resource to change the `replicas` value. 
++
+In this procedure, we change the `replicas` value for `my-topic` from 1 to 3. 
++
+.Kafka topic replication factor configuration
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 1
+  replicas: 3
+  # ...
+----
+
+. Apply the change to the `KafkaTopic` configuration and wait for the Topic Operator to update the topic.
+. Check the status of the `KafkaTopic` resource to make sure the request was successful: 
++
+[source,shell,subs="+quotes"]
+----
+oc get kafkatopics my-topic -o yaml
+----
++
+.Status for the replication factor change
+[source,shell,subs="+attributes"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 1
+  replicas: 1
+  # ...
+
+# ...
+status:
+  conditions:
+  - lastTransitionTime: "2024-01-18T16:13:50.490918232Z"
+    status: "True"
+    type: Ready
+  observedGeneration: 2
+  replicasChange: 
+    sessionId: 1aa418ca-53ed-4b93-b0a4-58413c4fc0cb # <1>
+    state: ongoing # <2>
+    targetReplicas: 3 # <3>
```

Comment:
Reading the above description where we say that we change .spec.replicas from 1 to 3, we should change .spec.replicas to 3, so that it matches .status.replicasChange.targetReplicas.